haiku-apps/heidi/heidi-1.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Native IDE for Haiku"
+DESCRIPTION="Heidi is a useful IDE for Haiku where coders can edit, \
+build, run, and debug their code projects all in one window."
+HOMEPAGE="https://github.com/waddlesplash/Heidi"
+COPYRIGHT="2014 Austin Cavalier, Kacper Kasper"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="ec676d2b45043b7a70d606f5fe2dbf05f3c45ed1"
+SOURCE_URI="https://github.com/waddlesplash/Heidi/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="dca3b020f36e83d0dc83b10b1362b73c6f2240605122fefbfc554e49ad70df8c"
+SOURCE_FILENAME="Heidi-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Heidi-$srcGitRev"
+
+# The software is in alpha version:
+# Cannot be distributed yet.
+ARCHITECTURES="!x86_gcc2 !x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="!x86"
```

Comment:
Oops,
```diff
 if [ "$targetArchitecture" = x86_gcc2 ]; then
-SECONDARY_ARCHITECTURES="!x86"
+SECONDARY_ARCHITECTURES="x86"
 fi
```

Reply:
```
# The software is in alpha version:
# Cannot be distributed yet
```
Oops, I had not seen these 2 lines.
If the app should not be distributed in binary form then using `?x86` (instead of `x86`) and `?x86_64` (instead of `x86_64`) would be enough.